CardiorespiratoryFitness/CardiorespiratoryFitness/Model/CRFHeartRateStep.swift

```diff
@@ -38,12 +38,9 @@ import UIKit
 open class CRFHeartRateStep : RSDActiveUIStepObject, RSDRestartableRecorderConfiguration {
 
     private enum CodingKeys : String, CodingKey {
-        case shouldSaveBuffer, cameraSettings, isResting, shouldDeletePrevious
+        case cameraSettings, isResting, shouldDeletePrevious
     }
     
-    /// Should the log file include the full pixel matrix or just the averaged value?
```

Comment:
Have we (i.e. Megha) decided we don't need this anymore even for algorithm development and/or validation?

Reply:
Yes, and checked with Megha and it's not used.